### PR TITLE
[stdlib] [SE-0031] update stdlib for SE-0031 (inout adjustment)

### PR DESCRIPTION
**Pull request by dduan**
The `inout` adjustment has been [merged](https://github.com/apple/swift/pull/1333). This is a follow-up that adjust position of `inout` in all Swift code.

### CODE REVIEW

**Comment by lattner** [comment]
Thanks!

**Comment by dduan** [comment]
@lattner thank you, sir!

File: test/IDE/complete_at_eof_in_call_no_newline_1.swift [hunk 1]
```diff
@@ -7,4 +7,4 @@
 // A-DAG: Pattern/ExprSpecific:               ['(']{#Int#})[#Void#]{{; name=.+$}}
 // A: End completions
 func f(x: Int) {}
-f(#^A^#
```
Comment (gribozavr) [inline test/IDE/complete_at_eof_in_call_no_newline_1.swift]: Please restore the absence of newline of EOF.  This is a regression test.

**Comment by gribozavr** [comment]
@dduan Sorry, but you changed meaning of some tests.  You added a line at EOF everywhere.  Some tests require that newline to be absent, they are testing that case in the compiler.

Could you audit your changes, and remove the newline where we didn't have it?

**Comment by dduan** [comment]
@gribozavr sure, feel free to revert this. I'll make another PR when these changes are done.

**Comment by dduan** [comment]
@gribozavr turns out sed is being *too* helpful in this case. Sorry.

**Pull request merged**